web/libs/datamanager/src/components/CellViews/index.js

```diff
@@ -15,3 +17,10 @@ export { StringCell as Text } from "./StringCell";
 export { VideoCell as Video } from "./VideoCell";
 export { ProjectCell as Project } from './ProjectCell';
 
+export function normalizeCellAlias(alias) {
+  // remove trailing separators to make `toStudlyCaps` safe
+  const safeAlias = alias.replace(/[-_\s]+$/g, '');
+    // remove trailing separators to make `toStudlyCaps` safe
+    .replace(/[-_\s]+$/g, '');
```

Comment:
with the applied replacement we wont need this anymore - infact as it is it would cause an error
```suggestion

```